File: kt/jvm/src/test/java/TestContext.java
import kt.context.Context;
import kt.context.ContextKt;
import org.junit.Test;

public class TestContext {


    @Test
    public void testInvokeKt() {
        Context c = new Context();
        System.out.println(ContextKt.px2dp(c, 10));
    }
}

File: kt/jvm/src/test/kotlin/kt/context/TestContext.kt
package kt.context

import org.junit.Test
import kotlin.test.assertEquals
import kotlin.test.assertTrue

class TestContext {

    @Test
    fun `test dp 2 px`() {
        val dp = with(Context()) {
            px2dp(100)
        }
        assertEquals(100f, dp)
    }
}